docs/lib/storage/fragments/ios/configureaccess.md

```diff
@@ -1,19 +1,26 @@
 ## Protected access
 
-Create an options object with the protected access level to restrict access for certain objects.
+Create an `Options` object with the `.protected` access level to restrict access for your key.
 
 ```swift
-let options = StorageDownloadDataRequest.Options(accessLevel: .protected)
-Amplify.Storage.downloadData(key: "myKey", options: options) { (event) in
+let dataString = "My Data"
+let data = dataString.data(using: .utf8)!
+let options = StorageUploadDataRequest.Options(accessLevel: .protected)
+Amplify.Storage.uploadData(key: "userProtectedKey", data: data, options: options) { (event) in
     // handle result
 }
 ```
 
-When uploading a file with `protected` access level, users can only read the file and only the user which created the file can delete it.
+When uploading a file with `.protected` access level:
```

Comment:
Is this defining what the protected access level is? The "when uploading" part makes it less clear.

Reply:
Chatted offline: need to reference the code sample above that we are uploading a file, and as a result of using the code above that the .protected access level is that owerns will have the ability to read/write/delete etc & other users will have read access (assuming they use the .protected flag and targetIdentityId